### PR TITLE
Add XML

### PR DESCRIPTION
**Pull request by himanshu1603**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/xml/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+created_by: World Wide Web Consortium
+display_name: XML
+released: 1996
+short_description: XML is subset of SGML(standard general markup language) used to create web page.
```
Comment (MikeMcQuaid) [inline topics/xml/index.md]: Space after `SGML`. I wonder if it's worth describing what it's used for normally (which is not related to web pages) like your longer description?

File: topics/xml/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+created_by: World Wide Web Consortium
+display_name: XML
+released: 1996
+short_description: XML is subset of SGML (standard general markup language) used to store and transport data.
```
Comment (jeffrafter) [inline topics/xml/index.md]: I think SGML should be expanded `Standard Generalized Markup Language`

File: topics/xml/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+topic: xml
+related: css, html, rss, xhtml
+url: http://www.w3.org/TR/rec-xml
```
Comment (jeffrafter) [inline topics/xml/index.md]: Not critical, but the link here redirects to `https://www.w3.org/TR/REC-xml/` which is https and has a different case. That said, this should probably be shortened to `https://www.w3.org/TR/xml/` as that is the "latest" rec link.

File: topics/xml/index.md (hunk 2 above)
Comment (MikeMcQuaid) [reply topics/xml/index.md]: - `general` to `generalized`

**Comment by MikeMcQuaid** [comment]
Great work again @himanshu1603!

> I 😍 XML!

😭

**Pull request merged**